src/libs/actions/Task.ts

```diff
@@ -1216,7 +1216,19 @@ function getTaskOwnerAccountID(taskReport: OnyxEntry<OnyxTypes.Report>): number
 /**
  * Check if you're allowed to modify the task - anyone that has write access to the report can modify the task, except auditor
  */
-function canModifyTask(taskReport: OnyxEntry<OnyxTypes.Report>, sessionAccountID: number): boolean {
+function canModifyTask(
+    taskReport: OnyxEntry<OnyxTypes.Report>,
+    sessionAccountID: number,
+    allowEditingConciergeTask?: boolean,
+    taskOwnerAccountID?: number,
+    taskAssigneeAccountID?: number,
```

Comment:
```suggestion
```
Do we have a use case that requires using this parameter? If the change comes from a previous update, I think we can consider remove this prop.